### DISCUSSION TITLE
Can’t get backgrounds to work

### DISCUSSION THREAD

**Discussion by bradykeithfisher** [karlcswanson/micboard#37] [Q&A]
I downloaded the desktop app for Mac, and everything is working great except I can’t get backgrounds to populate. I have loaded pictures in the background folder in the configuration directory, and restarted the server. Is there a trick to this?

**Comment by simonsaucy** [comment]
I cant get it to open on mac os Monterey and ideas?

**Comment by timgiannosa1** [comment]
Im having the same issue Im running MacOS sequoia

**Comment by landenm45-debug** [comment]
I am having the same issues.
